stable/rabbitmq/templates/configuration.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "rabbitmq.fullname" . }}-config
+data:
+  enabled_plugins: |
+      [rabbitmq_management,rabbitmq_peer_discovery_k8s].
+
+  rabbitmq.conf: |
+      ## Clustering
```

Comment:
I think this configuration should come from the values.yaml. 
Example: https://github.com/kubernetes/charts/blob/master/stable/mariadb/templates/configmap.yaml

Reply:
ok,  I have moved the main body of both config files, but left the user and password configuration as that is configure via other variables. you cant reference other variables via values.yaml